homeassistant/components/recorder/util.py

```diff
@@ -28,7 +28,7 @@ def session_scope(*, hass=None, session=None):
         if session.transaction:
             need_rollback = True
             session.commit()
-    except Exception as err:  # pylint: disable=broad-except
+    except Exception as err:
```

Comment:
Why this work? That is a broad exception

Reply:
It is, but pylint doesn't consider it an issue if the same block raises.